concourse/scripts/pipecleaner.py

```diff
@@ -137,6 +162,10 @@ def check_pipeline(self, data):
                         for i in item['config']['outputs']:
                             output_resources.add(i['name'])
 
+                    if 'run' in item['config']:
+                        if item['config']['run']['path'] in ['sh', 'bash', 'zsh']:
+                            self.shellcheck(item['config']['run']['path'], job['name'], item['task'], item['config']['run']['args'])
```

Comment:
As above, we should be doing something with the output of shellcheck rather than just spewing it at stdout.